SDL_GetMouseState.mediawiki

```diff
@@ -31,6 +32,8 @@ button), and <code>x</code> and <code>y</code> are set to the mouse cursor
 position relative to the focus window. You can pass NULL for either
 <code>x</code> or <code>y</code>.
 
+You may wanna use [[SDL_GetGlobalMouseState]]() for multi monitor setup, since x and Y drop to 0 when passing from 1 monitor to the next, thus you don't know which monitor the mouse is on...
```

Comment:
The coordinates are window relative for this function.

Reply:
SDL_GetGlobalMouseState() should return the extended buttons in the latest snapshot, can you verify?
https://www.libsdl.org/tmp/SDL-2.0.zip

Reply:
I downloaded unpacked and did:
./configure
make
sudo make install
in the unpacked directory, and it finished without problems. Then I completely rebuilt my application and it does the same with SDL_GetGlobalMouseState as it did before. Still only recognizes 3 of 5 buttons.

Reply:
Is the problem maybe that the XQueryPointer call here doesn't return extended button state?
https://github.com/libsdl-org/SDL/blob/main/src/video/x11/SDL_x11mouse.c#L403

Reply:
Yup, that's the problem...

Okay, try this?
https://www.libsdl.org/tmp/SDL-2.0.zip (updated file)

Reply:
This is the output of SDL_GetMouseState
00000:00:15.887217 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 1
00000:00:15.961068 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 0
00000:00:17.267401 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 2
00000:00:17.431823 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 0
00000:00:18.150384 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 4
00000:00:18.265462 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 0
00000:00:20.231536 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 8
00000:00:20.391225 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 0
00000:00:23.179676 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 16
00000:00:23.448986 [ Trace  1000 -->   Debug ]: X 364 Y 181 Buttons 0

Reply:
Just to confirm, I'm on Ubuntu 20.10 (a bit outdated...) Windowing system: X11